modules/rostests/kmtests/kmtest/support.c

```diff
@@ -159,8 +163,28 @@ KmtUnloadDriver(VOID)
 
     if (Error)
     {
+        ok(FALSE, "KmtStopService('%ls') failed! (Error = 0x%08lx)\n", TestServiceName, Error);
+
         // TODO
         __debugbreak();
```

Comment:
Should I remove these `__debugbreak();`?
Otherwise, what are these TODOs intended to become?

Reply:
I would assume this is something that is not finished, hence why someone wanted to break in here.

Reply:
Hopefully, @ThFabba, as author, shall answer these with more details.